pyaqueduct/client/client.py

```diff
@@ -309,6 +313,26 @@ def remove_tag_from_experiment(self, experiment_uuid: UUID, tag: str) -> Experim
         logging.info("Removed tag %s from experiment <%s>", tag, experiment_obj.title)
         return experiment_obj
 
+    def remove_files_from_experiment(self, experiment_uuid: UUID, files: List[str]) -> None:
+        """
+        Remove files from an experiment
+
+        Args:
+            experiment_uuid: UUID of experiment for which files has to be removed from.
+            files: List of file names to be removed.
+
+        """
+
+        remove_url = f"{self.url}/files/{str(experiment_uuid)}/delete_files"
+        try:
+            response = post(remove_url, timeout=self.timeout, json={"file_list": files})
+        except TransportError as error:
+            raise FileRemovalError("Couldn't remove files due to transport error.") from error
```

Comment:
What is transport error? Is it the one which is usually raised if server raised an exception? Then the reason is probably different, e.g. "Due to a server error: .."

Reply:
Yes, it is the server error and it includes connection errors too. I change transport to server.